spec/policies/event_policy_spec.rb

```diff
@@ -3,21 +3,65 @@
 RSpec.describe EventPolicy do
 
   let(:reviewer) { create(:user, :reviewer) }
+  let(:program_team) { create(:user, :program_team) }
   let(:organizer) { create(:organizer) }
   let(:admin) { create(:admin) }
   let(:event) { create(:event) }
 
   subject { described_class }
 
-  #permissions ".scope" do
-  #  pending "add some examples to (or delete) #{__FILE__}"
-  #end
+  permissions ".scope" do
```

Comment:
I think we can delete the pending permissions scope given you've got real permission testing below.